docs/lib/storage/fragments/ios/upload.md

```diff
@@ -32,14 +31,10 @@ do {
 
 _ = Amplify.Storage.uploadFile(key: fileNameKey, local: filename) { event in
     switch event {
-    case let .completed(data):
+    case let .success(data):
         print("Completed: \(data)")
-    case let .failed(storageError):
+    case let .failure(storageError):
         print("Failed: \(storageError.errorDescription). \(storageError.recoverySuggestion)")
-    case let .inProcess(progress):
-        print("Progress: \(progress)")
```

Comment:
can you add `progressListener` to this one too?

Reply:
Added